Tip 273 - Azure Key Vault and scripting initial commit

whaakman opened the pull request:
Added a tip on using Azure Key Vault Secrets in PowerShell

jongio commented on blog/tip273.md:
```diff
@@ -0,0 +1,67 @@
+
+:bulb: Learn more : [Microsoft Azure Key Vault](https://azure.microsoft.com/services/key-vault?WT.mc_id=azure-azuredevtips-micrum). 
+
+:tv: Watch the video : [Getting started with Azure Key Vault](https://azure.microsoft.com/nl-nl/resources/videos/azure-key-vault-developer-quick-start/).
```
Is locale required here?

stefanpetter replied:
I think "https://azure.microsoft.com/en-us/resources/videos/azure-key-vault-developer-quick-start/" is better here. The locale will be automagically determined (probably through the language settings of your browser)

whaakman replied:
Fixed both, thanks!

mbcrump commented:
hey @whaakman , can you bump the tip # up to 274 here? we already have a 273 and 274 is now the powershell one

whaakman commented:
> hey @whaakman , can you bump the tip # up to 274 here? we already have a 273 and 274 is now the powershell one

Hi @mbcrump, 275 then? :)

whaakman commented:
> > hey @whaakman , can you bump the tip # up to 274 here? we already have a 273 and 274 is now the powershell one
> 
> Hi @mbcrump, 275 then? :)

@mbcump Fixed and updated to 275. Also updated according to reviewers.

mbcrump commented:
thanks @whaakman !!!
